src/components/table/table_header_cell.test.tsx

```diff
@@ -92,15 +92,23 @@ describe('EuiTableHeaderCell', () => {
   });
 
   describe('sorting', () => {
-    it('is rendered with isSorted', () => {
+    it('renders with a sortable icon if `onSort` is passed', () => {
+      const { container } = renderInTableHeader(
+        <EuiTableHeaderCell onSort={() => {}}>Test</EuiTableHeaderCell>
+      );
+
+      expect(container.firstChild).toMatchSnapshot();
+    });
+
+    it('renders a sort arrow with isSorted', () => {
       const { container } = renderInTableHeader(
         <EuiTableHeaderCell isSorted>Test</EuiTableHeaderCell>
       );
 
       expect(container.firstChild).toMatchSnapshot();
     });
 
-    it('is rendered with isSortAscending', () => {
+    it('renders a sort arrow upwards isSortAscending', () => {
```

Comment:
just noticed a typo in my changes 😬 
```suggestion
    it('renders a sort arrow upwards with isSortAscending', () => {
```